xlators/features/marker/src/marker-quota.c

```diff
@@ -644,7 +644,7 @@ mq_get_set_dirty(xlator_t *this, loc_t *loc, int32_t dirty, int32_t *prev_dirty)
     return ret;
 }
 
-int32_t
+static void
 mq_mark_dirty(xlator_t *this, loc_t *loc, int32_t dirty)
 {
     int32_t ret = -1;
```

Comment:
nit: if you don't care about the value of ret, no point in initializing it to -1 here.

Reply:
It's convention to initialize all vars, so will leave it as-is to follow convention.